packages/graphback-codegen-resolvers/tests/snapshots/ApolloServiceResolverGeneratorTest.ts.md

```diff
@@ -4,7 +4,78 @@ The actual snapshot is saved in `ApolloServiceResolverGeneratorTest.ts.snap`.
 
 Generated by [AVA](https://ava.li).
 
-## Test snapshot config ts
+## Test snapshot resolvers js
+
+> Snapshot 1
+
+    {
+      custom: [
+        {
+          name: 'getLikedNotes',
+          output: `␊
+          exports.getLikedNotes = {␊
+            Query: {␊
+              getLikedNotes: (_, args, context) => {␊
+                // Implementation here␊
+              }␊
+            }␊
+          }␊
+          `,
+        },
+        {
+          name: 'likeNote',
+          output: `␊
+          exports.likeNote = {␊
+            Mutation: {␊
+              likeNote: (_, args, context) => {␊
+                // Implementation here␊
+              }␊
+            }␊
+          }␊
+          `,
+        },
+        {
+          name: 'index',
+          output: `const { getLikedNotes } = require('./getLikedNotes')␊
+          const { likeNote } = require('./likeNote')␊
+          ␊
+          exports.customResolvers = [getLikedNotes, likeNote]␊
+          `,
+        },
+      ],
+      index: `const { commentResolvers } = require('./generated/comment')␊
+      const { noteResolvers } = require('./generated/note')␊
+      ␊
+      const { customResolvers } = require('./custom')␊
+      ␊
+      exports.resolvers = [commentResolvers, noteResolvers, ...customResolvers]␊
```

Comment:
@ardatan - you have mentioned specific function from graphql toolkit that we can use here. What